src/pybind/mgr/nfs/cluster.py

```diff
@@ -41,6 +41,7 @@ def create_ganesha_pool(mgr: 'MgrModule') -> None:
         mgr.check_mon_command({'prefix': 'osd pool application enable',
                                'pool': POOL_NAME,
                                'app': 'nfs'})
+        log.debug(f"Successfully created nfs-ganesha pool {POOL_NAME}")
```

Comment:
Just an observation: [it's recommended to use Lazy Logging](https://medium.com/swlh/why-it-matters-how-you-log-in-python-1a1085851205) (in Dashboard module the linter complains about this).